### PR TITLE
Item class

### PR DESCRIPTION
**Pull request by DaisyLi233**

### CODE REVIEW

**Comment by wuhanyi70** [comment]
These entities are good, and we also need 2 more entities: employee and employer.

**Review by Robinrswanson — APPROVED** [review]
good work

File: src/main/java/TempDataStorage.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    public void updateQuantity(String serinum, int modified_quantity){
+        // modified_quantity can be both positive and negative!
+        Item value = this.inventory.get(serinum);
+        int quantity = value.getQuantity() + modified_quantity;
+        value.setQuantity(quantity);
+    }
+
+    public boolean checkQuantity(String serinum, int decreased_quantity) throws Exception {
```
Comment (wuhanyi70) [inline src/main/java/TempDataStorage.java:17]: Careful consideration for the negative quality.

**Review by wuhanyi70 — APPROVED** [review]
Add some basic entity classes.

**Pull request merged**